core/includes/install.core.inc

```diff
@@ -1832,6 +1839,7 @@ function _install_configure_form($form, &$form_state, &$install_state) {
   $form['admin_account']['account']['mail'] = array(
     '#type' => 'email',
     '#title' => st('E-mail address'),
+    '#description' => st("Notifications about pending security upadtes and site problems will be sent to this email address. It can be an address hosted on an external domain (such as a gmail.com or hotmail.com email)."),
```

Comment:
> `Notifications about pending security upadtes and site problems will be sent to this email address.`
> None of this is true, it needs to be removed.

Although this PR is closed, for the record this is actually true. The _site admin email_ is indeed used by update module. In `install.core.inc` we have this bit of code:

```
    // Add the site maintenance account's email address to the list of
    // addresses to be notified when updates are available, if selected.
    if ($form_state['values']['update_status_module'][2]) {
      config_set('update.settings', 'update_emails', array($form_state['values']['account']['mail']));
    }
```

So the administrator email is indeed used by default for security update emails.